app/__init__.py

```diff
@@ -22,16 +22,15 @@
 """
 
 from pyrogram import Client
-from config import API_ID, API_HASH, BOT_TOKEN, SYLTOKEN
+from config import API_ID, API_HASH, SESSION
 from .utils import Sylviorus
 from .db import LocalDb
 
-bot = Client("Sylviorus",
-             api_id=API_ID,
-             api_hash=API_HASH,
-             bot_token=BOT_TOKEN,
-             plugins=dict(root="{}/plugins".format(__name__)))
-
+bot = Client(
+      session_name=SESSION,
+      api_id=API_ID,
+      api_hash=API_HASH,
+      plugins=dict(root="{}/plugins".format(__name__))
+)
 
 ldb = LocalDb("reasons")
-SYL = Sylviorus(SYLTOKEN)
```

Comment:
why tho ?
it will mess up the whole system

Reply:
> why tho ? it will mess up the whole system

Hah